### DISCUSSION TITLE
Append extra data like pagination to the meta title

### DISCUSSION THREAD

**Discussion by daun** [aerni/statamic-advanced-seo#128] [Q&A]
Hi! I'm trying to figure out if it's possible to append extra data to the page's meta title, like the current page number or filter params. For example, a page with the title 'News' rendering `<title>News: Page 2</title>` when on the second page, or a collection mount with an optional filter param rendering `<title>News: Press</title>` when loaded with a `?filter=press` param. 

Is this currently doable?

**Comment by aerni** [comment]
You can use Antlers in the title field. So you could set it to `News: {{ get:filter }}`.

**Reply by daun** [reply]
It's a bit more complex and I'd like the editors not to deal with that part, so I'm looking to build the appended title somewhere in a controller or view composer and then append it to whatever the editor entered. Is that possible? Happy to send a PR your way if you think this would be useful. Thinking of some type of custom transformer or composer class that would take the existing SEO data right before rendering it, transform it, then pass it back to the addon.

**Reply by aerni** [reply]
Transformers sound like a fun idea. But you might already be able to achieve what you want with a [computed value](https://statamic.dev/computed-values#overview). You could create a computed field like `seo_computed_title` that takes in the `seo_title` and adds whatever you need. And then publish the addon's views and use the `seo_computed_title` instead of the `seo_title` for the `<title>` meta field.

**Reply by daun** [reply]
Oh nice, thanks for the pointer! I'll explore using computed values for this.
